sdk-generate-entity-models-maven-plugin/src/main/java/com/hpe/adm/nga/sdk/generate/GeneratorHelper.java

```diff
@@ -56,12 +56,18 @@ public static String camelCaseFieldName(final String fieldName, boolean theFirst
         int wordCounter = 0;
```

Comment:
You could also use "_+" to split on one or more underscores.
I'd personally replaceAll("_+", " ") then trim, then split on "\\s+" to handle the case where a field starts with underscores for some reason.
But catching the exception works too, I guess.